client/pages/Geo/GeoMap/GeoMapMenuMosaic/SatelliteSourceHeader/satelliteSourceHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import './satelliteSourceHeader.scss'
+import React, { useCallback } from 'react'
+
+import Icon from '@client/components/Icon'
+
+interface Props {
+  title: string
+  checked: boolean
+  isOpen: boolean
+  tabIndex: number
+  onCheckboxClick: () => void
+  onExpandClick: (isOpen: boolean) => void
+}
+
+const SatelliteSourceHeader: React.FC<Props> = ({
+  title,
+  checked,
+  isOpen,
+  tabIndex,
+  onCheckboxClick,
+  onExpandClick,
+}) => {
+  const handleExpandClick = useCallback(() => {
+    onExpandClick(!isOpen)
+  }, [isOpen])
+
+  return (
+    <div className="geo-map-menu-mosaic-select-header">
+      <div
+        className="geo-map-menu-mosaic-select-checkbox"
+        role="checkbox"
+        aria-checked={checked}
+        tabIndex={tabIndex}
+        onClick={onCheckboxClick}
+        onKeyDown={onCheckboxClick}
+      >
+        <div className={`fra-checkbox${checked ? ' checked' : ''}`} />
```

Comment:
we use classNames package for handling dynamic classNames:
https://www.npmjs.com/package/classnames

Not required, but just FYI :)

I think it would become something along the lines of (check this: not sure!)
`className={classNames('fra-checkbox', { checked })}`

Reply:
Okay, that's cool 👍 I added it